### PR TITLE
Use HW filter to only allow service requests to be passed on from the MCP2515.

### PR DESCRIPTION
**Pull request by aentinger**
These are:
- ExecuteCommand
- Access
- List
- NodeInfo.

### CODE REVIEW

**Review by pavel-kirienko — CHANGES_REQUESTED** [review]
Acceptance filter coalescence is slightly more complicated than that. Please use [`canardConsolidateFilters`](https://github.com/OpenCyphal/libcanard/blob/8ce8ed2de12b5d2421a39fb183a8ce1840e3dca1/libcanard/canard.h#L694) instead.

Also, if you are short on filters (as you appear to be), you may want to use [`canardMakeFilterForServices`](https://github.com/OpenCyphal/libcanard/blob/8ce8ed2de12b5d2421a39fb183a8ce1840e3dca1/libcanard/canard.h#L677) instead (notice the plurality). It will create a single filter that accepts all service transfers addressed to your node, and ignore those sent to any other node on the bus. If someone bothered sending a service request specifically to your node, then there's a good chance you should look into it and accept the request.

**Comment by aentinger** [comment]
Currently filters all messages  ... -.- possible an error with the hardware filter of the MCP2515.

Computer filter values for node ID **43** are:
```bash
[I] CAN Filter #1
        Ext. Mask :  2803F80
        Ext. ID   :  2001580
```

File: l3xz-leg-ctrl-firmware.ino [hunk 1]
```diff
@@ -283,6 +283,26 @@ void setup()
...
   mcp2515.begin();
   mcp2515.setBitRate(CanBitRate::BR_250kBPS_16MHZ);
+
+  CanardFilter const CAN_FILTER_SERVICES = canardMakeFilterForServices(node_id);
+  DBG_INFO("CAN Filter #1\n\r\tExt. Mask : %8X\n\r\tExt. ID   : %8X",
+           CAN_FILTER_SERVICES.extended_mask,
+           CAN_FILTER_SERVICES.extended_can_id);
+
+  /* Only pass service requests/responses for this node ID through to receive buffer #0. */
+  uint32_t const RXMB0_MASK = CAN_FILTER_SERVICES.extended_mask;
```
Comment (pavel-kirienko) [inline l3xz-leg-ctrl-firmware.ino:293]: Are you sure you're not supposed to set `MCP2515::CAN_EFF_BITMASK` here as well?

Comment (pavel-kirienko) [reply l3xz-leg-ctrl-firmware.ino:293]: Can you `candump` the bus, find your service request there, and ensure that its ID matches the filter?

Comment (aentinger) [reply l3xz-leg-ctrl-firmware.ino:293]: I found a bug in the MCP2515 driver, fixed in https://github.com/107-systems/107-Arduino-MCP2515/pull/86.

**Review by pavel-kirienko — APPROVED** [review]

**Pull request merged**